acceptance-tests/pom.xml

```diff
@@ -16,7 +16,7 @@
         <plugin-under-test.name>atlassian-bitbucket-server-integration</plugin-under-test.name>
         <jenkins.acceptance-test-harness.version>1.71</jenkins.acceptance-test-harness.version>
         <httpclient.version>4.5.10</httpclient.version>
-        <jenkins.version>2.176.4</jenkins.version>
+        <jenkins.version>2.190.1</jenkins.version>
```

Comment:
Can you do a quick check to see if our acceptance test library version is corresponding to this one?

Reply:
Thanks, good idea

Reply:
Checked this out, https://github.com/jenkinsci/acceptance-test-harness/blob/master/pom.xml looks like acceptance tests are using a much older version of Jenkins. Acceptance tests are green on bamboo so I think we're okay to go ahead with the version bump.